compiler/x/codegen/OMRCodeGenerator.cpp

```diff
@@ -228,12 +228,17 @@ OMR::X86::CodeGenerator::initializeX86(TR::Compilation *comp)
    // Determine whether or not x87 or SSE should be used for floating point.
    //
 
-#if defined(TR_TARGET_X86) && !defined(J9HAMMER)
+#if defined(TR_TARGET_X86)
+#if !defined(J9HAMMER)
    TR_ASSERT_FATAL(comp->compileRelocatableCode() || comp->isOutOfProcessCompilation() || comp->target().cpu.supportsFeature(OMR_FEATURE_X86_SSE2) == _targetProcessorInfo.supportsSSE2(), "supportsSSE2() failed\n");
 
    if (comp->target().cpu.supportsFeature(OMR_FEATURE_X86_SSE2) && comp->target().cpu.testOSForSSESupport())
```

Comment:
The SSE support check seems to be covered by the assertion.

Reply:
Yes.  This PR is about doing the minimal work to disable x87 by default in OMR to enable further refactoring.  The goal of the follow-on work to deprecate and remove the x87 code should be to smooth all of this out.

Reply:
@BradleyWood will wait for your approval if you're satisfied with Daryl's reasoning for this review. Thanks!

Reply:
Sure, go ahead. We will address in future work.